draft-packages/illustration/docs/IllustrationScene.stories.tsx

```diff
@@ -133,7 +133,7 @@ const IllustrationScenesTemplate: StoryFn<IllustrationScenesTemplateProps> = ({
   sceneComponents,
   ...restArgs
 }: IllustrationScenesTemplateProps) => {
-  const { isAnimated, loop, autoplay, alt = "", ...restProps } = restArgs
+  const { isAnimated, loop, autoplay, alt: _, ...restProps } = restArgs
```

Comment:
If it's optional, people would assume it still gets added to the component. I foresee people wondering why it's not being added and tests perhaps failing cause they are looking to the `alt` on the illustration...

Reply:
It does still get added to the `img`, if that's what you mean? This is just the story.